plugins/home/src/homePageComponents/WelcomeTitle/timeUtil.ts

```diff
@@ -22,7 +22,10 @@ import goodEvening from './locales/goodEvening.locales.json';
 // every time the user navigates.
 const greetingRandomSeed = Math.floor(Math.random() * 1000000);
 
-export function getTimeBasedGreeting(): { language: string; greeting: string } {
+export function getTimeBasedGreeting(language?: string[] | undefined): {
+  language: string;
```

Comment:
Would it be possible to export an enum with the existing keys in the locale files? If that's possible, we could use it as a language type to avoid string typos... That way people would be able to define the language array with just the keys that exist in the locale files. Another cool ability would be to accept a single string to get greetings always in the same language. Here is an example:
```tsx
import { WelcomeTitle, welcomeTitleLanguages } from '@backstage/plugin-home';

// Multiple languages with an array
<WelcomeTitle language={[welcomeTitleLanguages.English, welcomeTitleLanguages.Swedish]} />

// Single language with a string
<WelcomeTitle language={welcomeTitleLanguages.English} />

Reply:
Initially I have setup input for single language but than I received suggestion to convert it to array where we can input multiple as well as single language.

I am having difficulty in converting it to enum at the moment. will it be blocker in merging the PR?
@camilaibs

Reply:
No blocking, it was just a suggestion if possible.
If you later find an idea on how to do it, you are more than welcome to contribute to us again in a separate PR.